index.html

```diff
@@ -196,53 +174,57 @@ <h3 id="resource_wide_defaults">Resource-wide defaults</h3>
 
 
 <div class="req" id="bp_not_only_default">
-<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.</p>
+<p class="advisement">Specifications MUST NOT assume that a document-level default is sufficient.  Even If a resource-wide setting is available, specify field-based metadata to override the default.</p>
 </div>
 
 
 <p>Document level defaults, when combined with per-field metadata, can reduce the overall complexity of a given document instance, since the language and direction values don't have to be repeated across many fields. However, they do not solve all language or directionality problems, and so it must be possible to override the default on a string-by-string basis, where necessary.</p>
 
+<p>First-strong heuristics are ineffective when a default direction has been set for all strings, since  metadata overrides (intentionally) the value of the first-strong character. Therefore it is necessary to use explicitly provided field data to override the default. Even if an RLM character has been prepended to a string, the default metadata overrides it.</p>
```

Comment:
check spacing. double-space between "since" and "metadata overrides".

What this paragraph is saying is not quite true? First-strong heuristics are ineffective because they don't even apply when metadata is present. Characters like RLM still potentially have an effect, but they don't participate in setting the paragraph base direction. Perhaps instead say:

> First-strong heuristics are not applied to strings when the direction has been set externally using metadata. Even if an RLM character has been prepended to a string, the default metadata overrides it. Therefore it is necessary to use explicitly provided field data to override the default for strings whose base direction does not match the resource-wide default.

Reply:
I need to do another pass at the text to improve some other bits of text, too.  I'll come back to this then.  In the meantime, i'll merge this PR, so that changes of this kind become more clearly visible in the diff.